index-copy.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://unpkg.com/@tailwindcss/browser@4"></script>
+  <link rel="stylesheet" href="tailwind.css">
+  <title> </title>
+</head>
+<body class="bg-purple-200 flex items-center justify-center h-screen">
+  <div class="bg-white border-2 w-1/3 rounded-lg border-gray-300 flex flex-col items-center px-10 py-10">
+    <div class="flex  w-full justify-between items-center">
+      <img src="./assets/imgPerfil.png" alt="" srcset="">
+      <div class="flex flex-col w-1/2">
+        <h2 class="text-2xl font-bold ">Dulce Maria</h2>
+        <p class="font-light text-xs text-gray-400">Product Engineer</p>
+        <div class="flex gap-2 mt-1">
+          <img src="./assets/github.svg" alt="">
+          <img src="./assets/twitter.svg" alt="">
+          <img src="./assets/linkedin.svg" alt="">
+        </div>
+        <div class="bg-purple-600 rounded-md flex items-center justify-around px-3 py-1 text-white mt-2">
+          <img src="./assets/message.svg" alt="">
+          <button>Mensagem</button>
+        </div>
+      </div>
+    </div>
+    <hr class="my-4 border-gray-300 w-full">
+    <div class="flex justify-between w-full">
+      <div class="flex flex-col items-center">
+        <p class="font-bold text-purple-600 text-2xl">45</p>
+        <p class="font-light text-md text-gray-500">projetos</p>
```

Comment:
```suggestion
        <p class="font-light text-base text-gray-500">projetos</p>
```
acho que text-md não existe